### DISCUSSION TITLE
Can't click Sign in button

### DISCUSSION THREAD

**Discussion by tcm0116** [teslamate-org/teslamate#1151] [Q&A]
I've installed TeslaMate into Home Assistant OS running on a Raspberry Pi via the Portainer add-on to Home Assistant. I basically followed the easy setup instructions from the site below except that I'm using the MQTT add-on for Home Assistant.

https://satheesh.net/2019/09/28/teslamate-digitalocean-docker-step-by-step-installation-guide/

I'm able to access the TeslaMate login screen, but I can't click on the Sign in button after entering my Tesla account credentials. I don't see anything in the log that would indicate there's an issue.
```
18:11:21.484 [info] Already up,
18:11:56.506 [info] Version: 1.17.0,
18:11:56.616 [info] Running TeslaMateWeb.Endpoint with cowboy 2.7.0 at :::4000 (http),
18:11:56.621 [info] Access TeslaMateWeb.Endpoint at http://[nnn.nnn.nnn.nnn:4000],
18:11:57.833 [info] MQTT connection has been established,
18:11:59.777 [info] tzdata release in place is from a file last modified Wed, 11 Sep 2019 19:35:17 GMT. Release file on server was last modified Tue, 22 Dec 2020 23:35:21 GMT.,
18:12:08.101 [info] Tzdata has updated the release from 2019c to 2020e,
18:59:26.888 [info] GET /sign_in,
18:59:26.962 [info] Sent 200 in 73ms,
````
Any ideas what could prevent me from signing in?

**Comment by adriankumpf** [comment]
This could be because no WebSocket connection could be established. Try opening the browser console to see if any error messages are displayed in this regard.

**Reply by tcm0116** [reply]
I ended up installing it from this repository directly into Home Assistant, which is now working:

https://github.com/matt-FFFFFF/hassio-addon-repository
